docs/source/releasenotes/add-ons/3.0.rst

```diff
@@ -0,0 +1,42 @@
+.. _Version30ReleaseNotes:
```

Comment:
Koverse release notes should specify the addons version starting with 3.6.0. Maybe add a 3.6.rst and start the 3.6.0 release notes with the addon version?

Reply:
fixed it